ui/components/app/gas-timing/gas-timing.component.js

```diff
@@ -205,7 +205,10 @@ export default function GasTiming({
 }
 
 GasTiming.propTypes = {
-  maxPriorityFeePerGas: PropTypes.string,
-  maxFeePerGas: PropTypes.string,
+  maxPriorityFeePerGas: PropTypes.oneOfType([
```

Comment:
This was causing a warning to get generated during e2e tests, I thought I would just fix it so that they're easier to read in the future.